cli/src/semgrep/app/scans.py

```diff
@@ -25,7 +23,7 @@
 
 class ScanHandler:
     def __init__(self, dry_run: bool) -> None:
-        self.deployment_id, self.deployment_name = self._get_deployment_details()
+        self.deployment_id, self.deployment_name = None, None
```

Comment:
Now that these are initialized as None, don't they need to be set somewhere else?